src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/PowerShellHost.cs

```diff
@@ -536,7 +538,9 @@ private static void AddPathToEnvironment(string path)
             if (currentPaths.Add(path))
             {
                 var newPath = currentPath + Path.PathSeparator + path;
-                Environment.SetEnvironmentVariable("path", newPath, EnvironmentVariableTarget.Process);
+#pragma warning disable RS0030 // Do not use banned APIs (This add a path to the PATH environment variable just for the PowerShell console session)
```

Comment:
"just" for the PowerShell console session, and also every single other VS component running in devenv.exe 🤣